indico/web/client/styles/partials/_icons.scss

```diff
@@ -84,7 +84,9 @@ i[class^='icon-']::before {
  *      - icomoon.zip/selection.json to indico/web/static/fonts/icomoon/
  *      - Replace the icon definitions between the comments below with the ones found in icomoon.zip/style.css
           (pay attention to the indentation!)
-        - Replace any underscores with dashes in the css
+        - Replace any underscores with dashes in the css (icon-drag_indicator -> icon-drag-indicator)
+        - Add another colon to all pseudo-element selectors:
+          .icon-agreement:before should be .icon-agreement::before
```

Comment:
stylelint takes care of this, so people could just run `npx stylelint --fix indico/web/client/styles/partials/_icons.scss`

Reply:
I can add that to the instructions.